package/helper_scripts/check_for_k3s_etcd.sh

```diff
@@ -10,13 +10,17 @@ handle_error() {
 
 trap 'handle_error' ERR
 
+JOURNAL_LOG="${JOURNAL_LOG:-/var/log/journal}"
+if [[ "$(journalctl -D $JOURNAL_LOG --lines=0 2>&1 | grep -s 'No such file or directory' | wc -l)" -gt 0 ]]; then
+  JOURNAL_LOG=/run/log/journal
+fi
 
-if [[ "$(journalctl -D /var/log/journal -u k3s | grep -m1 'Managed etcd cluster initializing' | wc -l)" -gt 0 ]]; then
-    case $1 in 
+if [[ "$(journalctl -D $JOURNAL_LOG -u k3s | grep -m1 'etcd cluster' | wc -l)" -gt 0 ]]; then
```

Comment:
```suggestion
if [[ "$(journalctl -D $JOURNAL_LOG -u k3s | grep -m1 'Managed etcd cluster' | wc -l)" -gt 0 ]]; then
```

There are several logs in the K3s code that are `etcd cluster member list` or `joining etcd cluster` which this would catch.